assets/libraries/serverlessfw.rego

```diff
@@ -0,0 +1,37 @@
+package generic.serverlessfw
+
+import data.generic.common as common_lib
+
+resourceTypeMapping(resourceType, provider)= resourceTypeVal{
+    resourceTypeVal :=resourcesMap[provider][resourceType]
+}
+
+resourcesMap = {
+    "aws": {
+        "function": "AWS::Lambda",
+        "api": "AWS::ApiGateway",
+        "iam": "AWS::IAM"        
+    },
+    "azure":{
```

Comment:
I can be wrong, but I think it will be possible to run ARM queries for Crossplane trough `armTemplate`